js/scroller.js

```
@@ -0,0 +1,23 @@
+        const target = document.querySelector(this.getAttribute('href'));
+        const targetTop = target.getBoundingClientRect().top + window.pageYOffset;
+        const targetId = this.getAttribute('href');
+        history.pushState({ id: targetId, top: targetTop }, null, targetId);
+        target.scrollIntoView({
+            behavior: 'smooth'
+        });
+    });
+});
+
+window.addEventListener('popstate', function(e) {
+    if (e.state && e.state.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)